src/test/java/org/apache/bcel/generic/CountVisitor.java

```diff
@@ -0,0 +1,2012 @@
+        return visitLSHRCount;
+    }
+
+    public int getVisitLSTORECount() {
+        return visitLSTORECount;
+    }
+
+    public int getVisitLSUBCount() {
+        return visitLSUBCount;
+    }
+
+    public int getVisitLUSHRCount() {
+        return visitLUSHRCount;
+    }
+
+    public int getVisitLXORCount() {
+        return visitLXORCount;
+    }
+
+    public int getVisitMONITORENTERCount() {
+        return visitMONITORENTERCount;
+    }
+
+    public int getVisitMONITOREXITCount() {
+        return visitMONITOREXITCount;
+    }
+
+    public int getVisitMULTIANEWARRAYCount() {
+        return visitMULTIANEWARRAYCount;
+    }
+
+    public int getVisitNEWCount() {
+        return visitNEWCount;
+    }
+
+    public int getVisitNEWARRAYCount() {
+        return visitNEWARRAYCount;
+    }
+
+    public int getVisitNOPCount() {
+        return visitNOPCount;
+    }
+
+    public int getVisitPOPCount() {
+        return visitPOPCount;
+    }
+
+    public int getVisitPOP2Count() {
+        return visitPOP2Count;
+    }
+
+    public int getVisitPopInstructionCount() {
+        return visitPopInstructionCount;
+    }
+
+    public int getVisitPushInstructionCount() {
+        return visitPushInstructionCount;
+    }
+
+    public int getVisitPUTFIELDCount() {
+        return visitPUTFIELDCount;
+    }
+
+    public int getVisitPUTSTATICCount() {
+        return visitPUTSTATICCount;
+    }
+
+    public int getVisitRETCount() {
+        return visitRETCount;
+    }
+
+    public int getVisitRETURNCount() {
+        return visitRETURNCount;
+    }
+
+    public int getVisitReturnInstructionCount() {
+        return visitReturnInstructionCount;
+    }
+
+    public int getVisitSALOADCount() {
+        return visitSALOADCount;
+    }
+
+    public int getVisitSASTORECount() {
+        return visitSASTORECount;
+    }
+
+    public int getVisitSelectCount() {
+        return visitSelectCount;
+    }
+
+    public int getVisitSIPUSHCount() {
+        return visitSIPUSHCount;
+    }
+
+    public int getVisitStackConsumerCount() {
+        return visitStackConsumerCount;
+    }
+
+    public int getVisitStackInstructionCount() {
+        return visitStackInstructionCount;
+    }
+
+    public int getVisitStackProducerCount() {
+        return visitStackProducerCount;
+    }
+
+    public int getVisitStoreInstructionCount() {
+        return visitStoreInstructionCount;
+    }
+
+    public int getVisitSWAPCount() {
+        return visitSWAPCount;
+    }
+
+    public int getVisitTABLESWITCHCount() {
+        return visitTABLESWITCHCount;
+    }
+
+    public int getVisitTypedInstructionCount() {
+        return visitTypedInstructionCount;
+    }
+
+    public int getVisitUnconditionalBranchCount() {
+        return visitUnconditionalBranchCount;
+    }
+
+    public int getVisitVariableLengthInstructionCount() {
+        return visitVariableLengthInstructionCount;
+    }
+
+    private int visitAALOADCount = 0;
```

Comment:
Don't initialize values to their default values.
Group all instance variable declarations at the top of the file, see the rest of the code base.

Reply:
Updated, please check.